modules/030-cloud-provider-aws/images/cloud-controller-manager/patches/1.31/002-non-type-lb.patch

```diff
@@ -238,11 +235,11 @@ index 8f9884c..280caf1 100644
 +}
 +
  // ensureLoadBalancerv2 ensures a v2 load balancer is created
- func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBalancerName string, mappings []nlbPortMapping, instanceIDs, subnetIDs []string, internalELB bool, annotations map[string]string) (*elbv2.LoadBalancer, error) {
+ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBalancerName string, mappings []nlbPortMapping, instanceIDs, discoveredSubnetIDs []string, internalELB bool, annotations map[string]string) (*elbv2.LoadBalancer, error) {
```

Comment:
needs check  this patch because we change argument but not change variable in the function